examples/consumer/main.go

```diff
@@ -180,6 +187,32 @@ RETRY:
 	}
 }
 
+// listSubscriptions lists all subscriptions
+// and returns true if there are any subscriptions
+// NOTE for v1 this returns locally stored subscriptions even when subscriptions are deleted remotely
+func listSubscriptions() bool {
+	url := &types.URI{URL: url.URL{Scheme: "http",
+		Host: apiAddr,
+		Path: fmt.Sprintf("%s%s", apiPath, "subscriptions")}}
```

Comment:
No need to use Sprintf for this. 
```suggestion
		Path: (apiPath+"subscriptions")}}
```